presto-main/src/main/java/io/prestosql/server/ServerMainModule.java

```diff
@@ -191,6 +182,7 @@ protected void setup(Binder binder)
         configBinder(binder).bindConfig(FeaturesConfig.class);
 
         binder.bind(SqlParser.class).in(Scopes.SINGLETON);
+        SqlParserOptions sqlParserOptions = new SqlParserOptions();
         binder.bind(SqlParserOptions.class).toInstance(sqlParserOptions);
```

Comment:
```
binder.bind(SqlParserOptions.class).in(Scopes.SINGLETON);
``` 

instead of:

```
SqlParserOptions sqlParserOptions = new SqlParserOptions();
binder.bind(SqlParserOptions.class).toInstance(sqlParserOptions);	        
```

Reply:
I cannot because in the line below there a method called on this object o_O